### PR TITLE
Change and add links to the Cloud

### PR DESCRIPTION
**Pull request by dureuill**
- add cloud link in banner
- add utm to existing links following https://github.com/meilisearch/integration-guides/issues/277#issuecomment-1592054536

### CODE REVIEW

**Review by Kerollmops — APPROVED** [review]
Looks good to me @dureuill 🙏
bors merge

**Comment by curquiza** [comment]
@dureuill 
Thanks for the PR!
Don't put me as reviewer, put @gillian-meilisearch instead 😉 I still see all the PRs of the engine team

**Comment by dureuill** [comment]
Noted, @curquiza :+1:

(tbh I just added the "recommended reviewers" suggested by GH without giving it much more thought)

File: README.md [hunk 1]
```diff
@@ -60,7 +61,7 @@ You may also want to check out [Meilisearch 101](https://www.meilisearch.com/doc
 
 ## ☁️ Meilisearch cloud
 
-Let us manage your infrastructure so you can focus on integrating a great search experience. Try [Meilisearch Cloud](https://meilisearch.com/pricing) today.
+Let us manage your infrastructure so you can focus on integrating a great search experience. Try [Meilisearch Cloud](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=engine&utm_medium=meilisearch) today.
```
Comment (gmourier) [inline README.md:64]: Why do we choose a different message from the one added to all integration README.md files?

Comment (dureuill) [reply README.md:64]: I did not change that message it was written by @davelarkan

Comment (davelarkan) [reply README.md:64]: Indeed the initial message was added when the Cloud first opened for public release. We can change it now if we want something more appropriate?

Comment (dureuill) [reply README.md:64]: I personally don't have an opinion one way or the other

Comment (gmourier) [reply README.md:64]: Please go with the one added to the integration so we have something consistent everywhere (it serves the same purposes).

Thanks!

Comment (dureuill) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -58,13 +59,13 @@ For basic instructions on how to set up Meilisearch, add documents to an index,
...
-## ☁️ Meilisearch cloud
+## ⚡ Supercharge your Meilisearch experience
 
-Let us manage your infrastructure so you can focus on integrating a great search experience. Try [Meilisearch Cloud](https://meilisearch.com/pricing) today.
+Say goodbye to server deployment and manual updates with [Meilisearch Cloud](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=engine&utm_medium=meilisearch). No credit card required.
 
 ## 🧰 SDKs & integration tools
 
-Install one of our SDKs in your project for seamless integration between Meilisearch and your favorite language or framework!
+Install one of our SDKs in your proj``ect for seamless integration between Meilisearch and your favorite language or framework!
```
Comment (gmourier) [inline README.md:68]:
```suggestion
Install one of our SDKs in your project for seamless integration between Meilisearch and your favorite language or framework!
```

**Review by gmourier — COMMENTED** [review]
Thanks @dureuill 🚂

**Review by Kerollmops — APPROVED** [review]
Beautiful 🤩 I love what you've done there!
bors merge

**Pull request merged**